src/sentry/api/authentication.py

```diff
@@ -443,9 +443,10 @@ def authenticate_token(self, request: Request, token_str: str) -> tuple[Any, Any
                         # I soft launch this by only logging the error for now
                         # raise AuthenticationFailed("Unauthorized organization access")
                         logger.info(
-                            "Token has access to organization %s but wants to get access to organization %s",
+                            "Token has access to organization %s but wants to get access to organization %s: %s",
```

Comment:
would it be easier to find this log if the %s fields were logged in an `extra` dict?

Reply:
Doesn't matter much because I'm deleting it on Monday.